wpeverest/everest-forms
# Enhancement - Past Date disable

### All Submissions:


### Changes proposed in this Pull Request:
Previously there was no option to disable previous date. This PR enhance this feature.

### How to test the changes in this Pull Request:

1.  Go to the Form Builder.
2.  Drag and Drop the date field.
3.  Enable Past Date disable option.
4.  Verify it is working or not.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Past Date disable.
